### ISSUE TITLE
Guidance seems unexpected

### ISSUE THREAD

**Issue by brettf** [alexy201/awsdocsgpt#2]
I just started using the example website and entered in a couple example prompts, but the answers which came back were unexpected. 
> One Example: "How can I use the C IoT SDK to send messages to IoT Core that then land in an S3 bucket?"

..then directs me to using GreenGrass. 

It seems that either the IoT C SDK might not be included in training, or something else is amiss and needs tuning.

**Comment by alexy201** [comment]
Yes, this seems to be one of the limitations with traditional vector embeddings + semantic search. During the creation of this tool, the language-specific SDKs were not scraped which may lead to the error you mentioned. We will continually update our database of knowledge over time—thanks for bringing this issue up!
